### ISSUE TITLE
Enhancement: Wiki

### ISSUE THREAD

**Issue by cathalgarvey** [hughperkins/tf-coriander#41]
Now that `tf-coriander' has reached a state of relative usefulness, it might be helpful to have a wiki to collect information that others have gathered on how to put it to use.

E.g., I identified a version of Keras that I believe to be ~compatible with the version of Tensorflow that tf-coriander represents, and that's information I'm happy to share with people.

Common pitfalls and how to solve them falls into the same category of "tidbits worth sharing".

Basically, anything that applies specifically to using `tf-coriander`, either due to the older Tensorflow base, or to the OpenCL nature, or to peculiarities of Coriander, could be thrown on a Wiki. Just a thought. :)

**Comment by hughperkins** [comment]
Hi Cathal,

Ok sounds good.  What wiki do you have in mind?  I've enabled the github wiki on this repo, if that is what you are thinking of?  https://github.com/hughperkins/tf-coriander/wiki

**Comment by cathalgarvey** [comment]
That's perfect: minimal overhead and keep communal docs close to the code (eg, in the source tree!).

I'll find some time this weekend to throw up some docs. :)

On 3 June 2017 00:35:56 GMT+01:00, Hugh Perkins <notifications@github.com> wrote:
>Hi Cathal,
>
>Ok sounds good.  What wiki do you have in mind?  I've enabled the
>github wiki on this repo, if that is what you are thinking of? 
>https://github.com/hughperkins/tf-coriander/wiki
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/hughperkins/tf-coriander/issues/41#issuecomment-305932036

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by hughperkins** [comment]
k, awesome! :-)
